internal/pkg/util/email/sendinblue.go

```diff
@@ -0,0 +1,68 @@
+package email
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"strings"
+	"text/template"
+)
+
+var apiURL = "https://api.sendinblue.com/v3"
+
+const sendEmailTemplate = `{
+  "sender":{
+    "name": "Beem-auth",
+    "email":"no-reply@beem.com"
+  },
+  "to": [
+    {"email":"{{.Recipient}}"}
+  ],
+  "subject":"{{.Subject}}",
+  "htmlContent":"{{.Content}}"
+}`
+
+type sendinblue struct {
+	apiKey   string
+	template *template.Template
+}
+
+func NewSendinblue(apiKey string) *sendinblue {
```

Comment:
Can't we just initialize this in main so you don't need this as a variable in services? Just a call to `email.Sendmail()` instead of a special instance seems fine for our usecase.

Added benefit is that you can instantiate as a mock, dev, or production instance based on the environment

Reply:
After some experimentation, it's possible, however it makes mocking the email interface impossible. If we'd like to mock email in our tests this way, we'd need a struct anyway.